pycon/templates/finaid/receipt_form.html

```diff
@@ -0,0 +1,65 @@
+{% extends "site_base.html" %}
+
+{% load bootstrap_tags %}
+{% load compress %}
+{% load static %}
+{% load i18n %}
+{% load boxes_tags %}
+
+{% block head_title %}Receipt Form{% endblock %}
+
+{% block body_id %}receipt_form{% endblock %}
+
+{% block page_title %}Receipt Application{% endblock %}
+
+{% block body %}
+    {% if form.errors %}
+        <div class="alert">
+            <p class="text-error">
+                There were errors in your form, please correct them and submit again.
+            </p>
+        </div>
+    {% endif %}
+
+    <form method="POST" action="" enctype="multipart/form-data" class="form-horizontal">
+        {% csrf_token %}
+        {% if form.errors and not form.non_field_errors %}
+          <div class="alert alert-error">
+            {% blocktrans count counter=form.errors.items|length %}Please correct the error below.{% plural %}Please correct the errors below.{% endblocktrans %}
```

Comment:
I don't think I've tried this before. It looks like the example in the Django docs, but doesn't match the prose documentation as I'm reading it. Have you tested this? I'm curious which is right.